### DISCUSSION TITLE
add option to replace Randomised string

### DISCUSSION THREAD

**Discussion by k-Jithesh** [Innofactor/XrmToolBox.BulkDataUpdater#163] [Ideas]
Similar to Fixed String, Token etc, can there be another option to add random generated string. This would allow the tool to work as bulk anonymizer as well.

**Comment by rappen** [comment]
Great idea!
I will add this to the Calculate / XRM Token.
I can't really give you a date, sorry.
Using simple calculate like this gets you some "random" so far...
`<system|now|fff>`

**Comment by rappen** [comment]
Created is this issue: https://github.com/rappen/Rappen.XTB.Helper/issues/14
